apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerCommandTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Data;
+using System.Linq;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public class SpannerCommandTests
+    {
+        [Fact]
+        public void CommandTypeSupportsOnlyCommandText()
+        {
+            var command = new SpannerCommand();
+            Assert.Throws<NotSupportedException>(() => command.CommandType = CommandType.StoredProcedure);
+            Assert.Throws<NotSupportedException>(() => command.CommandType = CommandType.TableDirect);
+        }
+
+        [Fact]
+        public void UpdateRowSourceNotSupported()
+        {
+            var command = new SpannerCommand();
+            Assert.Throws<NotSupportedException>(() => command.UpdatedRowSource = UpdateRowSource.Both);
+            Assert.Throws<NotSupportedException>(() => command.UpdatedRowSource = UpdateRowSource.FirstReturnedRecord);
+            Assert.Throws<NotSupportedException>(() => command.UpdatedRowSource = UpdateRowSource.OutputParameters);
+        }
+
+        [Fact]
+        public void PrepareNoop()
+        {
+            var command = new SpannerCommand();
+            command.Prepare();
```

Comment:
nah I could check all state, but its a bit overkill.  prepare is actually a no-op and is ignored.